### PR TITLE
parallel building

### LINKED ISSUE #40: Travis CI is taking a long time to compile the code
The average build time is around 27mins or so, it would have been better if we could investigate faster CIs and move the build function to there and keep the Content Delivery with Travis (metapackage pushing to gh-pages). What are your thoughts?

### PR DESCRIPTION
**Pull request by meets2tarun**
Fixes #40 
### Short Description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,23 +3,27 @@ language: generic
...
 sudo: required
 
-script:
-  - ./build.sh
-  - 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+jobs:
+  include:
+    - script: ./build.sh
+    - script: 'if [ "$TRAVIS_PULL_REQUEST" = "false" ]; then bash ./scripts/aptRepoUpdater.sh; fi'
+    - stage: GitHub Release
```
Comment (abishekvashok) [inline .travis.yml]: This must be done after build.sh and not here.

Comment (abishekvashok) [inline .travis.yml]: Don't make deploying parallel.

Comment (meets2tarun) [inline .travis.yml]: Gtihub Release although happens after build.sh completed not before.

Comment (abishekvashok) [inline .travis.yml]: Then why are you parallising it: `Github Release although happens after build.sh completed not before`

Comment (meets2tarun) [inline .travis.yml]: abhi
Deploy build only get started when the both the test cases successfully complete

**Comment by meets2tarun** [comment]
@xeon-zolt Please review this, if everything is ok, we can merge this.

**Pull request merged**